src/server/scripts/EasternKingdoms/ScarletEnclave/chapter1.cpp

```diff
@@ -434,14 +435,7 @@ class npc_eye_of_acherus : public CreatureScript
                         Talk(TALK_CONTROL, owner);
                     }
                     me->SetDisableGravity(false);
-                    DoCast(me, SPELL_EYE_FLIGHT);
-                }
-
-                if (movementType == POINT_MOTION_TYPE && pointId == POINT_EYE_FALL)
-                {
-                    me->SetDisableGravity(true);
-                    me->SetControlled(true, UNIT_STATE_ROOT);
-                    _events.ScheduleEvent(EVENT_MOVE_START, 5000);
+                    me->CastSpell(me, SPELL_EYE_FLIGHT);
```

Comment:
DoCastSelf

Reply:
... using the format
```cpp
DoCastSelf(SPELL_EYE_FLIGHT);
```
(if I recall correctly)